docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+
+The ``-s | --sensor`` option may be used if only one of the two catalogues need to be downloaded / updated. Specifying one of the sensors of either family is sufficient (i.e., LT05 for Landsat or S2B for Sentinel-2).
+
+Parameterization
+----------------
+
+Basic syntax
+""""""""""""
+
+FORCE Level-1 CSD is parameterized with four mandatory arguments. These make up the core of the request and always need to be provided, while all remaining arguments are optional. The basic structure is visualized in the image below.
+
+.. figure:: img/tutorial-l1csd-syntax.jpg
+
+Mandatory arguments
+"""""""""""""""""""
+
+.. code-block:: none
+
+  metadata-dir
+  Directory where the metadata catalogues (csv file) are stored
+
+  level-1-datapool
+  An existing directory, your files will be stored here
+
+  queue
+  Downloaded files are appended to a file queue, which is needed for the
+  Level 2 processing. The file doesn't need to exist. If it does exist,
+  new lines will be appended on successful ingestion
+
+  area of interest
+  (1) user-supplied coordinates of your study area:
+      The polygon must be closed (first X/Y = last X/Y). X/Y must be given as
+      decimal degrees with negative values for West and South coordinates.
+      Either specify the path to a file, or the coordinates on the command line.
+      If in a file, provide one coordinate per line.
+      If on the command line, provide a comma separated list.
+  (2) a shapefile (point/polygon/line). On-the-fly reprojection is provided,
+      but using EPSG4326 is recommended.
+  (3) grid identifier:
+      Landsat: Path/Row as "PPPRRR". Make sure to keep leading zeros:
+        correct: 181034, incorrect: 18134
+      Sentinel-2: MGRS tile as "TXXXXX". Make sure to keep the leading T before
+        the MGRS tile number.
+      You can either give the path to a file, or give the IDs on the command line.
+      If in a file, provide one ID per line.
+      If on the command line, provide a comma separated list.
+
+The first three arguments specify the local directories for the metadata catalogues, the level 1 datapool (download directory), and the processing queue. We define our area of interest with the fourth argument. FORCE Level 1 CSD offers quite some flexibility in how the aoi can be specified. Options ``(1)`` and ``(2)`` will intersect the provided geometries with the tiling scheme of Landsat and/or Sentinel-2, while option ``(3)`` the third option can be used if we already know which Landsat path/row or Sentinel-2 tiles we need. If you want to specify coordinates or a list of grid identifiers on the command line, keep in mind that the maximum length of command line calls is limited and varies by system! So your lists may just be cut off and in the worst case you won't notice before it's too late. You may be better off storing them in a text file. Make sure this file uses UNIX end of line characters and provide one coordinate pair or grid identifier per line.
```

Comment:
please avoid these long one-liners

Reply:
tiling scheme --> reference system (WRS-2/MGRS)
I really would not want to term any of these as a proper tiling scheme...

Reply:
typo: while option ``(3)`` the third option

Reply:
Block starting with "If you want to..." could be moved to a .. warning:: directive

Reply:
some open questions here, especially (3)